deploy/charts/rook-ceph/templates/resources.yaml

```diff
@@ -7104,7 +7104,7 @@ spec:
                     activeCount:
                       description: The number of metadata servers that are active.
                       format: int32
-                      maximum: 10
+                      maximum: 50
```

Comment:
50 is as arbitrary as 10, but at least so much bigger I don't expect anyone will hit the limit intentionally